hrms/hr/workspace/expense_claims/expense_claims.json

```diff
@@ -247,7 +247,7 @@
   },
   {
    "hidden": 0,
-   "is_query_report": 0,
+   "is_query_report": 1,
```

Comment:
update `modified` timestamp for this workspace for the changes to get synced on migrate